### ISSUE TITLE
Aceitar somente letras do alfabeto na primeira palavra do nome do serviço

### ISSUE THREAD

**Issue by sahrosa** [servicosgovbr/editor-de-servicos#327]

**Comment by caduvieira** [comment]
e números?

**Comment by cv** [comment]
Qual o motivo para essa restrição? Algo técnico?

**Comment by sahrosa** [comment]
Sim! Um serviço foi cadastrado chamado de "2-via-de-carteira-de-habilitacao" e aí a lista de serviços começa a aumentar assim:

<img width="734" alt="screen shot 2015-09-24 at 10 03 34 pm" src="https://cloud.githubusercontent.com/assets/2932644/10090635/243b9464-6308-11e5-90ab-5bfacf6fbfbe.png">

Isso é um problema porque vai criando uma complexidade inesperada e torna o nome dos serviços inacessíveis. Esse serviço, se lido num leitor de tela falaria "dois via de carteira de habilitação".

**Comment by caduvieira** [comment]
Uma segunda via de algo não seria uma etapa de um serviço maior como "emissão de carteira" ?

Ou a pessoa registraria "segunda-via..."

**Comment by sahrosa** [comment]
Este é um outro ponto importante, @caduvieira. O que é, e o que não é um serviço :)
